aldryn_client/utils.py

```diff
@@ -155,6 +155,10 @@ def is_windows():
     return sys.platform == 'win32'
 
 
+def is_linux():
+    return sys.platform in ('linux', 'linux2')
```

Comment:
Are these two values enough?

(here is how twisted does it https://github.com/twisted/twisted/blob/f074ba3d5083aa1503abcf194aece327e7f84805/twisted/python/runtime.py#L136-L143)